spec/requests/iiif/auth/v2/probe_service_spec.rb

```diff
@@ -388,5 +402,21 @@
                                               })
     end
   end
+
+  describe '#parse_uri' do
```

Comment:
This seems like abusing request tests.  If we need to test this, can you extract this to a service and test it separately?

Reply:
How about something like https://github.com/sul-dlss/stacks/pull/1090 as an alternative way of testing this fix.